docs/users/overpass-api-guide.md

```diff
@@ -0,0 +1,318 @@
+
+## Tips and Best Practices
+
+### Rate Limiting
+
+Overpass API has rate limits:
+
+- **Avoid rapid-fire queries**: Use the `pulse` input to manually trigger refreshes
+- **Use reasonable timeouts**: `[timeout:25]` is usually sufficient
+- **Public mirrors**: Use alternative endpoints if needed:
+  - `https://overpass.kumi.systems/api/interpreter`
+  - `https://overpass.openstreetmap.fr/api/interpreter`
+
+### Bounding Box Size
+
+- **Small areas**: Overpass works best for city-block to city-scale queries
+- **Large bbox warning**: Queries over large regions may timeout
+- **Recommended size**: < 0.1° × 0.1° for complex features, < 1° × 1° for simple queries
+
+### Query Performance
+
+1. **Filter early**: Use specific tags to reduce result size
+2. **Limit geometry**: Use `out center;` for simple points instead of `out geom;`
+3. **Test queries**: Use [overpass-turbo.eu](https://overpass-turbo.eu/) to test before implementing
+4. **Be specific**: Narrow queries with precise tags run faster
+
+### Data Quality
+
+- **OSM tags**: Check [OSM Wiki](https://wiki.openstreetmap.org/wiki/Map_features) for tag definitions
+- **Data freshness**: Usually updates within minutes of OSM edits
+- **Missing data**: OSM coverage varies by region - urban areas generally have better data
+- **Tag variations**: Multiple tags may represent similar features (e.g., `building=yes` vs `building=house`)
+
+## Error Handling
+
+The `OverpassOp` automatically handles common errors:
+
+- **Timeout errors**: Returns empty GeoJSON FeatureCollection with console warning
+- **HTTP errors**: Throws error with status code
+- **Invalid responses**: Catches and reports fetch errors
+
+If you see errors, try:
+
+1. Reducing the bounding box area
+2. Simplifying the query (fewer tags/elements)
+3. Increasing the timeout: `[timeout:60]`
+4. Using a different endpoint
+
+## Differences from Overture Maps
+
+| Feature | Overpass API (OSM) | Overture Maps |
+|---------|-------------------|---------------|
+| **Data Source** | Live OpenStreetMap | Pre-processed OSM + other sources |
+| **Freshness** | Real-time (minutes) | Monthly releases |
+| **Query Language** | Overpass QL | SQL (via DuckDB) |
+| **Scale** | City/region scale | Global scale |
+| **Performance** | Rate limited, smaller queries | Large queries, pre-indexed |
+| **Use Case** | Current POIs, routing | Analytics, bulk processing |
+| **Data Model** | OSM tags (flexible) | Structured schema (consistent) |
+
+## When to Use Overpass vs Overture
+
+**Use Overpass API when:**
+
+- You need the latest OSM data
+- Querying specific, smaller areas
+- Working with OSM-specific tags and relations
+- Building tools that interact with OSM community data
+
+**Use Overture Maps when:**
+
+- Querying large geographic areas
+- Need consistent global schema
+- Performing analytics on bulk data
+- Don't need real-time updates
```

Comment:
We could probably remove this. The overviews at the top basically say the same thing